src/index.html

```diff
@@ -14,6 +14,7 @@
   </head>
   <body>
     <h1 class="logo">MA</h1>
+
```

Comment:
The link tag is referencing a SCSS file (`main.scss`). Browsers cannot directly interpret SCSS files. You should compile the SCSS file into a CSS file and link to the resulting CSS file instead.